### ISSUE TITLE
Noetic release?

### ISSUE THREAD

**Issue by felixvd** [ROBOTIS-GIT/dynamixel-workbench#324]
As reported in #308, the melodic-devel branch builds on Noetic. What about releasing the binaries? The same goes for dynamixel-workbench-msgs.

**Comment by ROBOTIS-Will** [comment]
@felixvd 
I'm on it. Thanks!

**Comment by OTL** [comment]
Hi, I came here to say +1.

**Comment by ROBOTIS-Shibata** [comment]
@ROBOTIS-Will 
You know that there are situations where ROS2 users still have to use ROS1 at the same time.
The Ubuntu major version of ROS2 user is 20.04. In other words, it is Noetic in ROS1.

**Comment by ROBOTIS-Will** [comment]
@ROBOTIS-Shibata 
Yes, once I release the package, it'll include all the recent ROS versions.

**Comment by ROBOTIS-Will** [comment]
Hi all! 
I'm sorry about the delay in Noetic release, but it is happening now.
https://github.com/ros/rosdistro/pull/30347

**Comment by yun-goon** [comment]
This issue appears to be resolved, so it will be closed.
